pkg/controller/machine/controller.go

```diff
@@ -25,14 +25,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
+	"os"
```

Comment:
standard imports should always be in the first group
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Fixed -- i'll try to figure out how to automate this.